about.html

```diff
@@ -35,8 +35,8 @@
         <ul class="menu">
           <li><a href="searchcriteria.html">Search Librar.io</a></li>
           <li><a href="index.html">Book Journal</a></li>
+          <li><a href="saved.html">Saved Books</a></li>
```

Comment:
I don't think we need another page for saved searches. It can be done on the book search page

Reply:
Dont you think we need another page to display the saved books?